apps/ledger-live-desktop/electron-builder.yml

```diff
@@ -55,7 +55,6 @@ nsis:
   uninstallerIcon: build/windows/uninstaller.ico
   uninstallerSidebar: build/windows/uninstallerSidebar.bmp
 
-includeSubNodeModules: true
```

Comment:
Why do we not need this anymore ?

Reply:
This option is now completely removed in the latest versions of electron-builder.